iml-agent/src/rpm.rs

```diff
@@ -0,0 +1,25 @@
+use regex::Regex;
+
+use std::process::Output;
+
+use crate::{agent_error::ImlAgentError, cmd::cmd_output};
+
+fn parse(output: Output) -> Result<bool, ImlAgentError> {
```

Comment:
Should this be parsing to something instead of returning a result of bool?

Reply:
This is what we came up with @jgrund initially, but I can change it.

What do you have in mind? `Result<YumQueryResult, ImlAgentError>` where
```rust
enum YumQueryResult {
  Installed,
  NotInstalled,
}
```
?

Reply:
I don't think we gain anything by using an enum over a bool here.